source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/navigation.tsx

```diff
@@ -4,23 +4,15 @@ import { Menu } from 'semantic-ui-react';
 import { NavLink } from 'react-router-dom';
 import { ROLE } from '@data/models/role';
 import { RequireRole } from '@ui/components/authorization';
-import { withTranslations, i18nProps } from '@lib/i18n';
+import { useTranslations } from '@lib/i18n';
 import { getCurrentOrganizationId } from '@lib/current-organization';
+import { useUserTasksForCurrentUser } from '~/data/containers/resources/user-task/list';
 
 export interface IProps {
   closeSidebar: () => void;
 }
 
-interface MenuItem {
-  name: string;
-  to: string;
-  onClick: (e) => void;
-  exact?: boolean;
-  className?: string;
-  target?: string;
-}
-
-const MenuItem = ({ onClick, className, ...otherProps }: MenuItem) => {
+const MenuItem = ({ onClick, className, ...otherProps }: any) => {
```

Comment:
props removed, because most of this relied on the "otherProps", and there is no good way to handle this without semantic-ui providing prop interfaces for every component (they don't)